src/pages/auth/login/login.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
```

Comment:
UI suggestion: the login form is not in align
![Screenshot 2022-03-29 at 6 17 39 PM](https://user-images.githubusercontent.com/79623546/160614572-6159ca3d-6cb0-486b-8bbd-5101583950e7.png)